src/AccessibilityInsights/MainWindow.xaml.cs

```diff
@@ -704,10 +704,30 @@ private static void RestoreConfigurationAsync(/*Action callback = null*/)
                 var serializedConfigsDict = appConfig.IssueReporterSerializedConfigs;
                 Dictionary<Guid, string> configsDictionary = JsonConvert.DeserializeObject<Dictionary<Guid, string>>(serializedConfigsDict);
                 configsDictionary.TryGetValue(selectedIssueReporterGuid, out string serializedConfig);
-                BugReporter.RestoreConfigurationAsync(serializedConfig);
+                await BugReporter.RestoreConfigurationAsync(serializedConfig).ConfigureAwait(true);
+                UpdateMainWindowConnectionFields();
```

Comment:
Is this part of code updating UI? if so, we need to call it with Dispatcher to run on UI thread.

Reply:
Discussed offline where we decided to go with a dispatcher call despit configureawait(true)